corehq/form_processor/models.py

```diff
@@ -1498,7 +1502,8 @@ def _from_form(cls, case, xform, transaction_type):
                 sync_log_id=xform.last_sync_token,
                 server_date=xform.received_on,
                 type=transaction_type,
-                revoked=not xform.is_normal
+                revoked=not xform.is_normal,
+                details=FormSubmissionDetail(xmlns=xform.xmlns).to_json()
```

Comment:
Will this need to be back-filled? If not, will there be repeaters in existence that expect to have the `xmlns` detail, but it will be missing?

Reply:
I haven't back-filled this. As it currently stands there is nothing that depends on the XMLNS being present.